crates/turborepo-lib/src/run/global_hash.rs

```diff
@@ -84,9 +84,25 @@ pub fn get_global_hash_inputs<'a, L: ?Sized + Lockfile>(
             }
         };
 
+        // This is a bit of a hack to ensure that we don't crash
+        // when given an absolute path on Windows. We don't support
+        // absolute paths, but the ':' from the drive letter will also
+        // fail to compile to a glob. We already know we aren't going to
+        // get anything good, and we've already logged a warning, but we
+        // can modify the glob so it compiles. This is similar to the old
+        // behavior, which tacked it on to the end of the base path unmodified,
+        // and then would produce no files.
+        #[cfg(windows)]
+        let windows_global_file_dependencies: Vec<String> = global_file_dependencies
```

Comment:
outputs at least are working fine. inputs crashes in both Rust and Go codepsth so it’s possible it would also be a problem, but not sure. We can probably merge this PR as is and bring back inputs in the test case to see how to handle that

Reply:
Yes, filed a ticket for our own `Glob` type, akin to our path types, to handle this.